buildpacks/cf.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: build.knative.dev/v1alpha1
 kind: BuildTemplate
 metadata:
-  name: buildpack
+  name: cloudfoundry
```

Comment:
Should we just remove this build template? Or maybe rename it to `cloudfoundry-buildpack`?

Reply:
The CNB builders don't support the Cloud Foundry buildpacks yet.